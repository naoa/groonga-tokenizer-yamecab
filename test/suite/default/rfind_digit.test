register tokenizers/yamecab

config_set tokenizer-yamecab.parse_limit 5

tokenize TokenYaMecab "12345678"
