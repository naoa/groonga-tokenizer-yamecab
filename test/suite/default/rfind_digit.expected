register tokenizers/yamecab
[[0,0.0,0.0],true]
config_set tokenizer-yamecab.parse_limit 5
[[0,0.0,0.0],true]
tokenize TokenYaMecab "12345678"
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "12345",
      "position": 0,
      "force_prefix": false
    },
    {
      "value": "678",
      "position": 1,
      "force_prefix": false
    }
  ]
]
